### DISCUSSION TITLE
Radegast v2.41

### DISCUSSION THREAD

**Discussion by cinderblocks** [cinderblocks/radegast#26] [Announcements]
## What's Changed
* Update LibreMetaverse to v2.0.9
* EventQueue handling fixed - fixes teleporting also
* Non-ASCII char fix for password hashing
* Make chat log saving location consistent with other viewers

## Known issues
* Updater still not responding (server side fix)
* Images do not decode on some systems
* Mesh deformations in 3D mode are inconsistent with SL viewer

**Full Changelog**: https://github.com/cinderblocks/radegast/compare/v2.40...v2.41

<hr /><em>This discussion was created from the release <a href='https://github.com/cinderblocks/radegast/releases/tag/v2.41'>Radegast v2.41</a>.</em>

**Comment by Venus-Drake** [comment]
[Radegast.log](https://github.com/cinderblocks/radegast/files/11487981/Radegast.log)
Brand new v2.41 fresh install log.

**Reply by Venus-Drake** [reply]
RLV teleport works if using global location, not local, where the TP seem to not find the region. Not sure if is because is a two words name region or not.
